.github/workflows/CI.yaml

```diff
@@ -178,3 +178,19 @@ jobs:
 
     - name: conformance
       run: TEST=true PYTHONPATH=. pytest test/registry/conformance_tests.py -s -vv
+
+  commit-check:
+    name: Commit Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: commit check
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: |
+            ^(.*):\s*(.*)\nPROJQUAY.*$
```

Comment:
I'd say if we're going to enforce that, put the JIRA ticket on the same line as the header.
i.e. `^(.*):\s*(.*)\s\(PROJQUAY-[0-9]+\)$`

Reply:
i like that idea